web/main.js

```diff
@@ -23,9 +23,26 @@ Vue.filter('timeOfDay', TimeFormatters.formatTimeOfDay);
 
 Vue.config.productionTip = false;
 
+const vuetify = new Vuetify({
+  theme: {
+    options: {
+      customProperties: true,
+    },
+    themes: {
+      light: {
+        default: '#272727',
```

Comment:
add a comment here about colours, to help devs understand how to use these throughout the application.